### PR TITLE
Allow ETag in trailers (#262)

### PR DESCRIPTION
**Pull request by reschke**

### CODE REVIEW

File: draft-ietf-httpbis-semantics-latest.xml [hunk 1]
```diff
@@ -9331,6 +9331,11 @@ Sun Nov  6 08:49:37 1994         ; ANSI C's asctime() format
    &MUST; mark the entity-tag as weak by prefixing its opaque value
    with "W/" (case-sensitive).
 </t>
+<t>
+   The Etag field can appear in the trailer section as well (see
+   (<xref target="trailer.fields"/>). Note however that recipients might
+   ignore it when it is not sent as header field.
```
Comment (mnot) [inline draft-ietf-httpbis-semantics-latest.xml]: as _a_ header field.

File: draft-ietf-httpbis-semantics-latest.xml [hunk 2]
```diff
@@ -9331,6 +9331,11 @@ Sun Nov  6 08:49:37 1994         ; ANSI C's asctime() format
    &MUST; mark the entity-tag as weak by prefixing its opaque value
    with "W/" (case-sensitive).
 </t>
+<t>
+   The Etag field can appear in the trailer section as well (see
+   (<xref target="trailer.fields"/>). Note however that recipients might
```
Comment (mnot) [inline draft-ietf-httpbis-semantics-latest.xml]: seems like a comma or two is missing

Comment (reschke) [reply draft-ietf-httpbis-semantics-latest.xml]: I do see an extra "(" that needs to go, but where do you miss a comma?

Comment (mnot) [reply draft-ietf-httpbis-semantics-latest.xml]: Note, however, that...

**Pull request merged**